README.md

```diff
@@ -17,15 +17,16 @@ python3 setup.py install --user
 pip3 install standup-notes
 ```
 ### Bash Setup
-For a 1 time execution, use
+
+To have bash completion, find where the bash_completion.d file is in your system by running the following command
```

Comment:
what's bash_completion.d? These are instructions on finding standup-notes.bash.

Reply:
That is the folder where the bash-completion script goes. Changes the instruction so that it says "where the standup-notes.bash file is"